package.json

```diff
@@ -27,15 +27,23 @@
     "eslint": "^9.12.0",
     "@eslint/js": "^9.12.0"
   },
-  "bundleDependencies": [],
   "license": "BSD",
   "engines": {
     "node": ">=0.8.2"
   },
   "scripts": {
-    "dev": "node ./tiddlywiki.js ./editions/tw5.com-server --listen",
-    "test": "node ./tiddlywiki.js ./editions/test --verbose --version --build index",
+    "start": "node ./tiddlywiki.js ./editions/multiwikiserver --mws-load-plugin-bags --build load-mws-demo-data --mws-listen",
+    "build:test-edition": "node ./tiddlywiki.js ./editions/test --verbose --version --build index",
+    "test:multiwikiserver-edition": "node ./tiddlywiki.js ./editions/multiwikiserver/ --build load-mws-demo-data --mws-listen --mws-test-server http://127.0.0.1:8080/ --quit",
+    "mws-add-user": "node ./tiddlywiki.js ./editions/multiwikiserver --build load-mws-demo-data --mws-listen --build mws-add-user --quit",
+    "test": "npm run build:test-edition && npm run test:multiwikiserver-edition",
     "lint:fix": "eslint . --fix",
     "lint": "eslint ."
+  },
+  "dependencies": {
+    "@playwright/test": "^1.47.2",
+    "better-sqlite3": "^9.4.3",
+    "node-sqlite3-wasm": "^0.8.10",
+    "playwright": "^1.47.2"
   }
```

Comment:
better sqlite3 version does not install on my Windows machine using Node.js 22.10.0
Version 11.5.0 which is latest does install.

Reply:
better-sqlite3 v11.0.0 -- dropped support for Node.js v21 -- see: https://github.com/WiseLibs/better-sqlite3/releases/tag/v11.0.0